doc/classes/InputEventKey.xml

```diff
@@ -89,21 +91,26 @@
 		</member>
 		<member name="physical_keycode" type="int" setter="set_physical_keycode" getter="get_physical_keycode" enum="Key" default="0">
 			Represents the physical location of a key on the 101/102-key US QWERTY keyboard, which corresponds to one of the [enum Key] constants.
```

Comment:
Similar sentiment to a prior comment. For some reason I cannot suggest this in bulk.

```suggestion
			Represents the physical location of a key on the 101/102-key US QWERTY keyboard, which corresponds to one of the [enum Key] constants. Physical key codes meant for game input, such as WASD movement, where only the location of the keys is important.
```